optuna/study.py

```diff
@@ -939,6 +939,32 @@ def delete_study(
 def get_all_study_summaries(storage: Union[str, storages.BaseStorage]) -> List[StudySummary]:
     """Get all history of studies stored in a specified storage.
 
+    Example:
+
+        .. testsetup::
+
+            import os
+
+            if os.path.exists("example.db"):
+                raise RuntimeError("'example.db' already exists. Please remove it.")
+
+        .. testcode::
+
+            import optuna
+
+            def objective(trial):
+                x = trial.suggest_float("x", -10, 10)
+                return (x - 2) ** 2
+
+            study = optuna.create_study(study_name="example-study", storage="sqlite:///example.db")
+            study.optimize(objective, n_trials=3)
+
+            study_sammary = optuna.study.get_all_study_summaries(storage="sqlite:///example.db")[0]
```

Comment:
Just an idea. What do you think? We could make the `[0]` more clear and also demonstrate some types and properties involved.
```suggestion
            study_summaries = optuna.study.get_all_study_summaries(storage="sqlite:///example.db")
            assert len(study_summaries) == 1
            
            study_summary = study_summaries[0]
            assert study_summary.study_name == "example-study"
```

Reply:
Thank you for nice idea, I agree with you.